packages/components/src/alignment-matrix-control/index.tsx

```diff
@@ -58,25 +57,26 @@ export function AlignmentMatrixControl( {
 		id
 	);
 
-	const compositeStore = useCompositeStore( {
-		defaultActiveId: getItemId( baseId, defaultValue ),
-		activeId: getItemId( baseId, value ),
-		setActiveId: ( nextActiveId ) => {
+	const classes = clsx( 'component-alignment-matrix-control', className );
+
+	const setActiveId = useCallback<
+		NonNullable< React.ComponentProps< typeof Composite >[ 'setActiveId' ] >
+	>(
+		( nextActiveId ) => {
 			const nextValue = getItemValue( baseId, nextActiveId );
 			if ( nextValue ) {
 				onChange?.( nextValue );
 			}
 		},
-		rtl: isRTL(),
-	} );
-
-	const activeId = useStoreState( compositeStore, 'activeId' );
```

Comment:
Moved reading the store's state to get the `activeId`  to the `Cell` component, which can access the store via context.